sdk/tests/tutorials/valuation/test_identifier_mismatch.py

```diff
@@ -0,0 +1,279 @@
+from utilities import InstrumentLoader
+from utilities import TestDataUtilities
+
+
+class IdentifiersRecipe(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        # Create a configured API client
+        api_client = TestDataUtilities.api_client()
+
+        # Setup required LUSID APIs
+        cls.transaction_portfolios_api = lusid.TransactionPortfoliosApi(api_client)
+        cls.portfolios_api = lusid.PortfoliosApi(api_client)
+        cls.instruments_api = lusid.InstrumentsApi(api_client)
+        cls.aggregation_api = lusid.AggregationApi(api_client)
+        cls.quotes_api = lusid.QuotesApi(api_client)
+        cls.recipes_api = lusid.ConfigurationRecipeApi(api_client)
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+        # Setup test data from utilities
+        cls.test_data_utilities = TestDataUtilities(cls.transaction_portfolios_api)
+
+        # Load instruments - InstrumentLoader uses 'Figi'
+        instrument_loader = InstrumentLoader(cls.instruments_api)
+        cls.instrument_ids = instrument_loader.load_instruments()
+
+        # Setup test parameters
+        cls.effective_date = datetime(2019, 4, 15, tzinfo=pytz.utc)
+
+        # Setup scopes for tests
+        cls.recipe_scope = "TestIdentifiers"
+        cls.recipe_code = "SimpleQuotes"
+
+        # Set market data scope for quotes and recipes
+        cls.market_data_provider = "Lusid"
+        cls.market_data_scope = "Test-" + str(uuid.uuid4())
+
+        # Setup test portfolio
+        cls.portfolio_code = cls.test_data_utilities.create_transaction_portfolio(
+            TestDataUtilities.tutorials_scope
+        )
+        cls.setup_portfolio(cls.effective_date, cls.portfolio_code)
+
+    @classmethod
+    def tearDownClass(cls):
+        # Delete portfolio once tests are concluded
+        cls.portfolios_api.delete_portfolio(
+            TestDataUtilities.tutorials_scope, cls.portfolio_code
+        )
+
+    def create_aggregation_request(
+        self, inline_recipe
+    ) -> lusid.models.AggregationRequest:
+        """
+        Creates an aggregation request that aggregates valuation results based on
+        provided inline recipe or recipe identifiers when using an upserted one
+        :param ConfigurationRecipe inline_recipe: A configured recipe to be used inline
+        :return: AggregationRequest
+        """
+
+        return models.AggregationRequest(
+            inline_recipe=inline_recipe,
+            metrics=[
+                models.AggregateSpec("Instrument/default/Name", "Value"),
+                models.AggregateSpec("Holding/default/PV", "Proportion"),
+                models.AggregateSpec("Holding/default/PV", "Sum"),
+            ],
+            group_by=["Instrument/default/Name"],
+            effective_at=self.effective_date,
+        )
+
+    @classmethod
+    def setup_portfolio(cls, effective_date, portfolio_code) -> None:
```

Comment:
i think there is already a setup portfolio you can use, given this isn't important to this example I think its ok to use that here